srcpkgs/wasmtime/template

```diff
@@ -0,0 +1,37 @@
+# Template file for 'wasmtime'
+pkgname=wasmtime
+version=0.19.0
+revision=1
+archs="~armv*"
```

Comment:
judging from [this code](https://github.com/bytecodealliance/wasmtime/blob/0387169934e91722b8b6819773eacc6893718296/crates/runtime/src/traphandlers.rs#L158-L172) you referenced earlier, you want to set `archs="x86_64* i686* aarch64*"`

Reply:
Good point, done.